source/math.d.ts

```diff
@@ -0,0 +1,218 @@
+import type {NumberAbsolute, BuildTuple, And, Or, ArrayMax, ArrayMin, PositiveNumericStringGt} from './internal';
+import type {IsEqual} from './is-equal';
+import type {PositiveInfinity, NegativeInfinity, IsNegative} from './numeric';
+
+/**
+Return the result of `A + B`
+Note:
+- A or B can only support `-999` ~ `999`.
+- A and B can only be small integers(A < 1000, B < 1000).
+- if the result is negative, you can only get `number`.
+@example
+```
```

Comment:
All examples should include import statement.

Reply:
done